### PR TITLE
Replaced old buggy verification with built in verification in request_ms_auth

### PR DESCRIPTION
**Pull request by mrdeveloperdude**

### CODE REVIEW

File: app/latigo/scheduler/__init__.py [hunk 1]
```diff
@@ -73,17 +53,14 @@ def _prepare_task_queue(self):
...
-        else:
-            logger.info(msg)
+        auth_config = self.model_info_config.get("auth")
+        if auth_config:
+            auth_config["verify_on_startup"] = False
+            auth_session = MsRequestsSession(MsSessionConfig(**auth_config))
+            res, msg = auth_session.verify_auth()
+            if not res:
+                self._fail(f"{msg} for session:\n'{auth_session}'")
+            del auth_session
```
Comment (devProdigy) [inline app/latigo/scheduler/__init__.py]: could you please move this to function and reuse it here as well
https://github.com/equinor/latigo/pull/45/files#diff-ab28f04a938ce4fa424e4aee005621ddR114-R120

File: app/latigo/types.py [hunk 2]
```diff
@@ -38,8 +38,8 @@ def __str__(self):
         return f"TimeRange({rfc3339_from_datetime(self.from_time)} -> {rfc3339_from_datetime(self.to_time)})"
 
 
-LatigoSensorTag = SensorTag
-# LatigoSensorTag = namedtuple("LatigoSensorTag", ["name", "asset"])
+# LatigoSensorTag = SensorTag
+LatigoSensorTag = namedtuple("LatigoSensorTag", ["name", "asset"])
```
Comment (mrdeveloperdude) [inline app/latigo/types.py]: Without this I would get the following error:
```

$ make code-quality
flake8 --ignore=E731,W503,W504,E501,E265,C0301,W1202,W1203 --max-complexity 10 --exclude build,junk --exit-zero "/home/y8/Desktop/Projects/latigo/code_quality/app"
mypy --ignore-missing-imports ""/home/y8/Desktop/Projects/latigo/code_quality/../app""
/home/y8/Desktop/Projects/latigo/app/latigo/types.py:47: error: Variable "latigo.types.LatigoSensorTag" is not valid as a type
/home/y8/Desktop/Projects/latigo/app/latigo/time_series_api/sensor_data_provider.py:101: error: LatigoSensorTag? has no attribute "name"
/home/y8/Desktop/Projects/latigo/app/latigo/time_series_api/sensor_data_provider.py:105: error: LatigoSensorTag? has no attribute "asset"

```

File: app/latigo/gordo/misc.py [hunk 3]
```diff
@@ -1,33 +1,14 @@
-import typing
+from typing import Dict, Optional
 import logging
 import pprint
-import pandas as pd
-import requests
 import copy
-from datetime import datetime
```
Comment (mrdeveloperdude) [inline app/latigo/gordo/misc.py]: I meanwhile took the liberty of cleaning this up now that my IDE started working somewhat again

File: app/latigo/scheduler/__init__.py (hunk 1 above)
Comment (mrdeveloperdude) [reply app/latigo/scheduler/__init__.py]: It was the skinniest DRY fix of my career I think 😅 Hope you like it.

File: app/latigo/types.py (hunk 2 above)
Comment (devProdigy) [reply app/latigo/types.py]: this was raising an error when I ran a code, are you sure it's works with it?

Comment (mrdeveloperdude) [reply app/latigo/types.py]: hmm.. I didn't see any errors and it was working for a long time. What errors did you see?

File: app/latigo/time_series_api/sensor_data_provider.py [hunk 4]
```diff
@@ -90,14 +90,20 @@ def get_data_for_range(
         missing_id = 0
         completed = 0
         data: typing.List[typing.Dict] = []
-        if len(spec.tag_list) <= 0:
+        # This is here to make typing explicit to help mypy to not fail
```
Comment (mrdeveloperdude) [inline app/latigo/time_series_api/sensor_data_provider.py]: I had another go at this. Can you see if this resolves your problem? Just disabling mypy created other problems

**Pull request merged**